### DISCUSSION TITLE
🟠 Change Detection Bug | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#637] [Announcements]
# 🟠 Change Detection Bug | Angular Challenges

Challenge 32 is about debugging an application that has issue when change detection is triggered

https://angular-challenges.vercel.app/challenges/angular/32-bug-cd/

**Comment by amochalin** [comment]
Seems like in Angular 21 (21.1.0) the routerLinkActive directive has been changed and now the initial code for this challenge works fine.

**Reply by phhien203** [reply]
yep, it works fine from my side

**Reply by tomalaforge** [reply]
yes I suppose I should delete this challenge, doesn't make any sense now
